File: PRAK403-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h>

int main() {
    int bil_1, bil_2;

    scanf("%d %d", &bil_1, &bil_2);

    if (bil_1 > bil_2) {
        int bil_maksimal = bil_1;
        int bil_minimal = bil_2;
        while (bil_maksimal >= bil_minimal) {
            printf("%d %d", bil_1, bil_2);
            if (bil_maksimal > bil_minimal) {
                printf(" - ");
            }
            bil_maksimal--;
            bil_1--;
            bil_2++;
        }
    }
    else {
        int bil_maksimal = bil_2;
        int bil_minimal = bil_1;
        while (bil_maksimal >= bil_minimal) {
            printf("%d %d", bil_1, bil_2);
            if (bil_maksimal > bil_minimal) {
                printf(" - ");
            }
            bil_maksimal--;
            bil_1++;
            bil_2--;
        }
    }
    return 0;
}

File: PRAK403-2410817310014-INDRA_SURYADILAGA.py
bil_1, bil_2 = map(int, input().split())

if bil_1 >= bil_2:
  bil_maksimal = bil_1
  bil_minimal = bil_2
  while bil_maksimal >= bil_minimal:
    print(f"{bil_1}-{bil_2} ", end=" ")
    bil_maksimal = bil_maksimal - 1
    bil_1 = bil_1 - 1
    bil_2 = bil_2 + 1

else :
  bil_maksimal = bil_2
  bil_minimal = bil_1
  while bil_maksimal >= bil_minimal:
    print(f"{bil_1}-{bil_2} ", end=" ")
    bil_maksimal = bil_maksimal - 1
    bil_1 = bil_1 + 1
    bil_2 = bil_2 - 1